ArduPlane/Attitude.cpp

```diff
@@ -46,6 +46,9 @@ float Plane::get_speed_scaler(void)
         // no speed estimate and not armed, use a unit scaling
         speed_scaler = 1;
     }
+    if (!plane.ahrs.airspeed_sensor_enabled()  && plane.g2.flight_options & FlightOptions::SURPRESS_TKOFF_SCALING  &&  plane.flight_stage == AP_Vehicle::FixedWing::FLIGHT_TAKEOFF) { //scaling is surpressed during climb phase of automatic takeoffs with no airspeed sensor being used due to problems with inaccurate airspeed estimates
```

Comment:
sorry to nitpick, but I prefer bit checks to have () around them, like this:
(plane.g2.flight_options & FlightOptions::SURPRESS_TKOFF_SCALING)
should also break the long line so it is 3 lines

Reply:
like this?

Reply:
yep! thanks